Doc/license.rst

```diff
@@ -782,7 +782,7 @@ of the OpenSSL license here::
      * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
      * SUCH DAMAGE.
      *
-     * The licence and distribution terms for any publically available version or
+     * The licence and distribution terms for any publicly available version or
```

Comment:
This seems to be a copy of a license (https://www.openssl.org/source/license-openssl-ssleay.txt), so I don't think it should be changed here.